### ISSUE TITLE
Prometheus metric "disk_storage_available" always return zero

### ISSUE THREAD

**Issue by estevao90** [minio/minio#10066]
When you check the metric disk_storage_available in /minio/prometheus/metrics route, the value is always zero, even if have space available.

## Expected Behavior

Metric disk_storage_available should show the available disk size.

## Current Behavior

The metric always return zero.

## Steps to Reproduce

1. Start minio
```shell
docker run --rm -p 9000:9000 --name minio1 -e "MINIO_ACCESS_KEY=admin" -e "MINIO_SECRET_KEY=admin123" -e "MINIO_PROMETHEUS_AUTH_TYPE=public" minio/minio server /data1 /data2 /data3 /data4
```
2. Open the metrics URL: http://localhost:9000/minio/prometheus/metrics
3. The disk_storage_available is zero

## Context

This problem started in release RELEASE.2020-07-11T06-07-16Z

## Your Environment

* Version used (`minio --version`): 2020-07-14T19:14:30Z
* Server setup and configuration:
* Operating System and version (`uname -a`): Linux Fedora 5.7.7-100.fc31.x86_64 #1 SMP Wed Jul 1 20:37:05 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

**Comment by harshavardhana** [comment]
To be fixed in #10065

**Comment by harshavardhana** [comment]
Fixed in #10065
